cmd/ct-woodpecker/main.go

```diff
@@ -0,0 +1,203 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"path"
+	"syscall"
+	"time"
+
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/ct-woodpecker/monitor"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	// default -config value
+	configDefault = "test/config.json"
+	// default metrics listen host address
+	metricsDefault = ":1971"
+)
+
+var (
+	clk    clock.Clock = clock.Default()
+	logger *log.Logger = log.New(
+		os.Stdout,
+		path.Base(os.Args[0])+" ",
+		log.LstdFlags)
+	signalToName map[os.Signal]string = map[os.Signal]string{
```

Comment:
`syscall.Signal` has a `String` method, any reason not to use that?

Reply:
I cribbed this from Boulder's `cmd` package and didn't check if it was still required :angel: 

I'll replace it. Good catch!

Reply:
Fixed in 5a8edcf